packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -156,7 +156,8 @@ export const getPolymarketMarkets = async (params: MonitoringParams): Promise<Po
 
   const { markets: polymarketContracts } = (await request(params.graphqlEndpoint, query)) as any;
 
-  // Remove markets with endDate after 1 week from now.
+  // Remove markets with 1 week old endDate or more. So we only monitor markets that ended in the last week
+  // (or are still ongoing).
```

Comment:
This leaves us with ~400 markets that take ~3-5 min to get their ancillary data cached and then the bot start running fast (every 60 secs as configured)